docs/api/covidcast-signals/_source-template.md

```diff
@@ -72,3 +72,30 @@ backfill here.
 If the signal has specific licensing or sourcing that should be acknowledged,
 describe it here. Also, include links to source websites for data that is
 scraped or received from another source.
+
+## Licensing
+
+### For signals constructed by Delphi:
+
+This data is provided under the terms of the [Creative Commons Attribution
+license](https://creativecommons.org/licenses/by/4.0/). If you use this data,
+you must cite [Delphi](https://delphi.cmu.edu/) COVIDcast API, preferably by
```

Comment:
I think we should sync as a group on our approach to branding this. (Or maybe you already have and I just missed it.) I've seen the `covidcast` API endpoint referred to by a couple of different names, and I think it would be good to settle on one.

Is there a plan to separate COVIDcast from Epidata? I assume not, but correct me if I'm wrong. In my mind, "the COVIDcast API" is really just one particular endpoint (i.e. `covidcast`) of the Epidata API, and COVIDcast isn't an API in itself. I think it would be most accurate in this sentence to replace "Delphi COVIDcast API" with "Delphi Epidata API", or if that's not specific enough then maybe "Delphi Epidata API for COVIDcast".

There's also the matter of COVIDcast R/Python clients which use the Epidata API behind the scenes. I would call those "SDKs" or "libraries" interchangeably, and might suggest citing them separately as the "COVIDcast SDK" or "COVIDcast client library". As in:

> The `COVIDcast client library` provides a lot of functionality, including, in part, the ability to fetch COVID-19-related data from the `Delphi Epidata API`, which is also surfaced though interactive maps and charts in the `COVIDcast` web application.

@ryantibs @RoniRos @capnrefsmmat interested to hear your thoughts.

Reply:
Recently merged PR #266 uses "COVIDcast Epidata API" and "Epidata API (other epidemics)"

Reply:
Roni is, I think, a fan of branding it the "COVIDcast Epidata API", as done in #266. "COVIDcast" is a pretty good brand name, so I'd be happy to centralize on that language.

I don't think we need to mention client package citations on each data source page -- we can put that in their documentation.

Reply:
Actually, @capnrefsmmat, we may have misunderstood each other.  I strongly support unified branding of the EpiData database and API.  It has a bright future for multiple diseases.  The only way for the rest of the world to make sense of it is as a unified DB & API for all epidemiological time series, with COVID-related time series being one component.   
I agree that "COVIDcast" is a very good brand name for COVID properties (if I should say so myself!).  We can still use it as an endpoint, and as the name for COVID-related maps, apps, tools, etc.  But I think the database and API should be understood to be EpiData.  So I support David's initiative to fix all references, I think it's long overdue.